src/pages/SearchPage.tsx

```diff
@@ -139,20 +138,22 @@ function SearchPage({betas, personalDetails, reports, isSearchingForReports}) {
 
     /**
      * Reset the search value and redirect to the selected report
-     *
-     * @param {Object} option
      */
-    const selectReport = (option) => {
+    const selectReport = (option: OptionData) => {
         if (!option) {
             return;
         }
         if (option.reportID) {
             Navigation.dismissModal(option.reportID);
         } else {
+            if (!option.login) {
+                return;
+            }
             Report.navigateToAndOpenReport([option.login]);
```

Comment:
Maybe this way it will be more safe? What do you think?
```suggestion
            Report.navigateToAndOpenReport(option.login ? [option.login] : []);
```